### PR TITLE
Adding example page

### PR DESCRIPTION
**Pull request by line47**
See issue https://github.com/18F/federalist/issues/1332

### Screenshot
localhost_1337_content_examples 2

### CODE REVIEW

**Comment by line47** [comment]
@OpenGlobe @wslack if you all want to work from this PR that'd be good with me 👍

**Comment by wslack** [comment]
Can we give the images an outline and not align them to the bottom of each section? I think both the images and the text boxes should be vertically centered.

**Comment by wslack** [comment]
I will work on content after reading though another set of edits.

**Comment by line47** [comment]
Borders might look a little odd on the images as the small screen view is hanging off the side a bit @wslack 

### Screenshot
screen shot 2017-12-08 at 9 49 42 am

### What about just a border on the bottom?
screen shot 2017-12-08 at 9 51 29 am

**Comment by line47** [comment]
Updates after @OpenGlobe changes
localhost_1337_content_examples 4

**Comment by wslack** [comment]
@line47 can we bevel the corners of the bottom examples just a bit?

**Comment by line47** [comment]
@wslack are you referring to the thumbnails or?

**Comment by line47** [comment]
### Most recent screenshot

localhost_1337_content_examples_

**Comment by wslack** [comment]
I was referring to the 2x3 grid at the bottom. If it doesn't look good, we don't need them, but I like the softer effect on https://18f.gsa.gov/what-we-deliver/

**Comment by line47** [comment]
The same box shadow is applied but I think because it is on a dark background the slight shadow is lost a bit.

**Comment by line47** [comment]
@jseppi mind giving this a review?

File: views/content/examples.njk [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          <p>Vote.gov makes it easier for eligible voters participate in our democratic process by providing streamlined information on how to vote. Site visitors can register electronically in states that allow it, and download a printable application in a number of languages for those states where electronic registration isn't an option.</p>
+          <p><a class="usa-button" href="https://docs.google.com/forms/d/1iB8aW7c9r1QH3s8XElQCrnXRGjAiPUYpWG1CMeEqGIo/viewform">Learn more</a></p>
+        </div>
+      </div>
+  </div>
+    <div class="featured-site">
+    <div class="usa-grid">
+      <div class="usa-width-one-half">
+        <h2>College Scorecard</h2>
+        <p>The U.S. Department of Education uses Federalist to host College Scorecard.  Combined with an API server, College Scorecard provides important information on everything from financial aidto expected earnings.  Federalist's cloud infrastructure helps manage site traffic during spikes in visitors, making sure College Scorecard is always available. </p>
```
Comment (jseppi) [inline views/content/examples.njk]: `aidto` -> `aid to`

File: frontend/sass/_examples.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.featured-site {
+padding: 5rem 0;
```
Comment (jseppi) [inline frontend/sass/_examples.scss]: indentation

File: views/content/examples.njk [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        <div class="usa-width-one-half">
+          <img src="/images/example-sites/vote-gov.png"/>
+        </div>
+        <div class="usa-width-one-half">
+          <h2>Vote.gov</h2>
+          <p>Vote.gov makes it easier for eligible voters participate in our democratic process by providing streamlined information on how to vote. Site visitors can register electronically in states that allow it, and download a printable application in a number of languages for those states where electronic registration isn't an option.</p>
+          <p><a class="usa-button" href="https://docs.google.com/forms/d/1iB8aW7c9r1QH3s8XElQCrnXRGjAiPUYpWG1CMeEqGIo/viewform">Learn more</a></p>
+        </div>
+      </div>
+  </div>
```
Comment (jseppi) [inline views/content/examples.njk]: indentation could use a little work here

**Comment by line47** [comment]
pushed up a fix for the typo and code formatting updates

**Pull request merged**